sdk/nodejs/automation/localWorkspace.ts

```diff
@@ -1154,6 +1160,15 @@ export class LocalWorkspace implements Workspace {
             args.push("--remote-skip-install-dependencies");
         }
 
+        if (this.remoteExecutorImage) {
+            args.push("--remote-executor-image=" + this.remoteExecutorImage.image);
```

Comment:
In Go we have some checks that `image`, `credentials.username` and `credentials.password` are not empty if `remoteExecutoreImage` or `remoteExecutorImage.credentials` is set respectively.  Is it worth adding a check for that here as well?

Reply:
I assumed this was more because Go will give default values to these things if they're omitted, whereas in Node you'd at least have to write an empty string explicitly. I can add the check, but it's not something we do for any of the other options in NodeJS, so maybe it's not worth the effort?

Reply:
Ah cool, I guess we can just leave it then.  That's just my lack of knowledge of TypeScript :)